.travis.yml

```diff
@@ -26,6 +26,10 @@ script:
     - "if [[ $TRAVIS_PYTHON_VERSION != '3.2' ]]; then for f in *.rst; do rst-lint $f; done; fi"
 after_success:
     - coveralls
+matrix:
+    allow_failures:
+        - os: osx
```

Comment:
isn't that for ignoring errors? from osx? If we're not running on osx, why would we ignore errors? ;)